cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -633,12 +632,13 @@ def _tx_scripts_hashes(
     hashes_db: dict = {}
 
     for r in records:
-        shash = cluster_obj.get_policyid(script_file=r.script_file)
-        shash_rec = hashes_db.get(shash)
-        if shash_rec is None:
-            hashes_db[shash] = [r]
-            continue
-        shash_rec.append(r)
+        if r.script_file:
```

Comment:
Optional: Keeping code as flat as possible is good for readability, so
```
if not r.script_file:
    continue
```
is better.